src/components/Onboarding/Slide/TagsSlide.vue

```diff
@@ -108,6 +106,25 @@ export default {
         this.$emit("previousSlide");
       }
     },
+    updateTagsList(data) {
+      //rebuild the component to prevent array length problems
+      console.log(this.images_bis)
+      this.carousel_key = data.images.length
+      if (this.images_bis.length > 0) {
+        this.show_carousel = true
+        this.images_for_carousel = data.images
+      }
+    },
+    showFullScreenCarousel() {
+      this.show_form = false;
+      this.$store.dispatch('setFullTagPageOrigin', 'tags_slide')
+      this.show_full_tag_page = true;
+      console.log("on change")
+    },
+    exitFullScreen() {
+      this.show_form = true
+      this.show_full_tag_page = false
+    }
```

Comment:
When you open the fullscreen view, you setup the origin page. Is it the same when you close the full? Is it necessary to setup also a new origin in this case?

Reply:
No I don't do it. I don't think it is necessary, since the origin change is only decisive when you go to the fullTagView from tagsslide or year collection slide. The exit of full screen does not matter in this case.